site/gatsby-site/server/tests/utils.ts

```diff
@@ -38,30 +39,67 @@ export const seedCollection = async ({ name, docs, database = 'aiidprod', drop =
     }
 }
 
-interface AuthResponse {
-    access_token: string;
-    refresh_token: string;
-    user_id: string;
-}
+export const seedUsers = async (users: { userId: string, roles: string[] }[], { drop } = { drop: true }) => {
 
-export const login = async (username: string, password: string) => {
-
-    const response = await fetch(`https://services.cloud.mongodb.com/api/client/v2.0/app/${config.REALM_APP_ID}/auth/providers/local-userpass/login`, {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json'
-        },
-        body: JSON.stringify({
-            username,
-            password,
-        })
+    await seedCollection({
+        name: 'users',
+        database: 'customData',
+        docs: users,
+        drop,
     });
+}
 
-    if (!response.ok) {
-        throw new Error(`Error login in! \n\n ${await response.text()}`);
-    }
+export const makeRequest = async (url: string, data: { variables?: Record<string, unknown>, query: string }) => {
 
-    const data: AuthResponse = await response.json();
+    return supertest(url)
+        .post('/')
+        .set('Authorization', `Bearer dummyToken`)
+        .send(data);
+}
 
-    return data;
-}
\ No newline at end of file
+export interface Fixture<T> {
```

Comment:
Or perhaps this is the element that needs documentation, on how it is expected to be used with respect to programmatic field tests.